oc_dms_mirror/rest_api/app/routes.py

```
@@ -1,68 +1,92 @@
+        """
+        self.logger.info(f"POST {request.url_rule.rule} from [{request.remote_addr}] with payload: {request.get_json()}")
+        try:
+            component = request.json.get('componentId')
+            if not component:
+                return self.response_json(400, {"result": "componentId cannot be blank"})
+
+            gav_template = self.dms_mirror.get_component_config(component)
+            if not gav_template:
+                return self.response_json(404, {"result": f"gav template for component {component} not found"})
+        except Exception as _e:
+            self.logger.exception(_e)
```

Ruff (0.14.2)

Redundant exception object included in `logging.exception` call

(TRY401)